packages/suite/src/utils/suite/validation.ts

```diff
@@ -149,6 +150,15 @@ export const validateMin =
         }
     };
 
+export const validateMax =
```

Comment:
this is too staking specific. How about validateStakingMax and moving it to some staking eth utils?

Reply:
moved, https://github.com/trezor/trezor-suite/pull/14713/commits/03dbbc780d1e2c8bcce13f33e84bcfa328df7e64